pkg/agent/querier/querier.go

```diff
@@ -54,6 +55,7 @@ type agentQuerier struct {
 	proxier                  proxy.Proxier
 	networkPolicyInfoQuerier querier.AgentNetworkPolicyInfoQuerier
 	apiPort                  int
+	role                     types.AgentRole
```

Comment:
Are we going to get this role somewhere? Because I did not see the getter function.

Reply:
Thinking over, it is better to add the `nodeType` into NodeConfig, then we don't need it here. I would remove it from `agentQuerier` struct